docs/02-getting-started/02-setup/01-starcoin-usage.md

```diff
@@ -22,4 +22,83 @@ SUBCOMMAND:
 - help  Prints this message or the help of the given subcommand(s)
 
 
+## Console
 
+There are two ways to use starcoin's console, one is to enter the console at the same time as the node starts.
+
+
+The following command will start a dev node and enter the console:
+
+```shell
+starcoin -n dev console
+```
+
+The following command will start a barnard node and enter the console.
+
+```shell
+starcoin -n barnard console
+```
+
+If started this way, the console and the node are in the same process, and if the console exits, the node will also exit automatically.
+
+Another way is to connect to a started node via the console.
+
+
+### Start cli console
+
+It is assumed that your node directory is the default directory, if not please specify it with the -d parameter.
+
+Execute the following command to access the starcoin console.
+
+- To connect via the local IPC.
+
+This command is the same as the one that starts the node while entering the console. The command will automatically detect if there is an ipc file in the directory, and if so, it will automatically connect the node.
+
+``` shell
+starcoin -n barnard console
+```
+
+Or specify the ipc file explicitly. 
+
+``` shell
+starcoin --connect ~/.starcoin/barnard/starcoin.ipc console
+```
+
+Note: The path to the ipc file is different on Windows
+
+``` shell
+starcoin.exe --connect \\. \pipe\starcoin.ipc console
```

Comment:
Space is redundant.  \\.\pipe\starcoin.ipc

Reply:
fix at https://github.com/starcoinorg/starcoin-cookbook/pull/20